Makefile

```diff
@@ -491,33 +430,7 @@ openshift-ci-test-build: \
 	@tail -n"+3" "config/deploy/addons.managed.openshift.io_addoninstances.yaml" > "config/openshift/manifests/addoninstances.crd.yaml";
 
 .SECONDEXPANSION:
-# cleans the built image .tar and image build directory
-clean-image-cache-%:
-	@rm -rf ".cache/image/$*" ".cache/image/$*.tar"
-	@mkdir -p ".cache/image/$*"
```

Comment:
Let's have one mage-equivalent clean-image-cache-* target? With the help of this, we can call it as Dependency before the execution of every BuildBundle mage target for safer builds (https://github.com/openshift/addon-operator/pull/137#discussion_r801559908)

wdyt :)